app/main.py

```diff
@@ -1,2 +1,62 @@
+from typing import Any, Union
+
+from app.point import Point
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.length = 8
+        self.size = 0
+        self.hash_table = [(None,)] * self.length
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/210570172-bc919b85-b96a-4207-8227-b5daa255505e.png)
there is no resize func in ur dictionary